ui/selectors/multichain/networks.test.ts

```diff
@@ -206,15 +212,17 @@ describe('Multichain network selectors', () => {
         getMultichainNetworkConfigurationsByChainId(
           mockMultichainNetworkStateWithBitcoinSupportDisabled,
         ),
-      ).toStrictEqual({ ...mockEvmNetworksWithNewConfig });
+      ).toStrictEqual({
+        ...mockEvmNetworksWithNewConfig,
+        [SolScope.Mainnet]: mockNonEvmNetworks[SolScope.Mainnet],
```

Comment:
We shouldn't have this network being enabled here, since it was initially disabled?